tests/forte/data/data_pack_profiling_test.py

```diff
@@ -0,0 +1,448 @@
+# Copyright 2019 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+profiling test for data pack: using entry (0.2.0) and using new methods (>0.3.0).
```

Comment:
We probably shouldn't make profiling test to associate with specific versions. They should be runnable for any versions?

Reply:
Ok the entry interface is not changed so removed version related comments and added a brief overview.